client/components/public/imports/HomeIntro.jsx

```diff
@@ -34,31 +34,36 @@ export default class HomeIntro extends Component {
                 <br />
                 <Header size='medium'>What is the Puzzle Hunt?</Header>
                 <Segment basic size='large' className="no-padding">
-                  <p>The Great Puzzle Hunt is like a scavenger hunt adventure with puzzles. Teams of up to 6 people travel on foot to various locations solving a total of four hour-long puzzles gathering clues along the way to solve one final meta puzzle.</p>
-                  <p>These are no ordinary puzzles though! It will take a diverse set of skills and talents to solve them! Our mission is to celebrate everyone's talents and demonstrate knowledge comes in many forms. Everything is timed using your phone, QR codes, and this online system! So yes you can win :)</p>
+                  {/* <p>The Great Puzzle Hunt is like a scavenger hunt adventure with puzzles. Teams of up to 6 people travel on foot to various locations solving a total of four hour-long puzzles gathering clues along the way to solve one final meta puzzle.</p>
```

Comment:
if these are permanent changes, may as well delete the old bullets as well to reduce the number of commented lines we have

Reply:
I've deleted some of the comments that are likely permanent changes!